src/python/evaluation/evaluation_config.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+
+from src.python import MAIN_FOLDER
+from src.python.evaluation.support_functions import create_folder
+
+
+class ApplicationConfig:
+    def __init__(self, args):
+        self.tool_path = args.tool_path
+        self.output_format = args.format
+        self.data_path = args.data_path
+        self.traceback = args.traceback
```

Comment:
Parse default arguments from [`configure_arguments`](https://github.com/hyperskill/hyperstyle/blob/xlsx_run_tool/src/python/evaluation/xlsx_run_tool.py#L25) – would it be better to set default parameters in `__init__`? Thought it might be a sort of duplication

Reply:
It is better to use default values in the `configure_arguments` function and have this `__init__` function